app/controllers/register/trustees/individual/CheckYourAnswersIndividualContactDetailsController.scala

```diff
@@ -19,34 +19,59 @@ package controllers.register.trustees.individual
 import config.FrontendAppConfig
 import controllers.Retrievals
 import controllers.actions._
+import identifiers.register.trustees.IsTrusteeNewId
+import identifiers.register.trustees.individual.{TrusteeEmailId, TrusteePhoneId}
 import javax.inject.Inject
 import models.{Index, Mode}
 import navigators.Navigator
 import play.api.i18n.{I18nSupport, MessagesApi}
 import play.api.mvc.{Action, AnyContent}
 import services.UserAnswersService
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
-import utils.Enumerable
-import utils.annotations.TrusteesIndividual
+import utils.{AllowChangeHelper, CountryOptions, Enumerable, UserAnswers}
+import controllers.register.trustees.individual.routes._
+import viewmodels.AnswerSection
+import utils.checkyouranswers.Ops._
+import models.Mode.checkMode
+import views.html.check_your_answers
 
-import scala.concurrent.ExecutionContext
+import scala.concurrent.{ExecutionContext, Future}
 
 class CheckYourAnswersIndividualContactDetailsController @Inject()(val appConfig: FrontendAppConfig,
                                                                    val messagesApi: MessagesApi,
-                                                                   val userAnswersService: UserAnswersService,
-                                                                   val navigator: Navigator,
                                                                    authenticate: AuthAction,
                                                                    getData: DataRetrievalAction,
                                                                    allowAccess: AllowAccessActionProvider,
-                                                                   requireData: DataRequiredAction
+                                                                   requireData: DataRequiredAction,
+                                                                   implicit val countryOptions: CountryOptions,
+                                                                   allowChangeHelper: AllowChangeHelper
                                      )(implicit val ec: ExecutionContext) extends FrontendController with Retrievals with I18nSupport with Enumerable.Implicits {
 
   def onPageLoad(mode: Mode, index: Index, srn: Option[String]): Action[AnyContent] =
-    (authenticate andThen getData(mode, srn) andThen allowAccess(srn) andThen requireData) {
-      implicit request => NotImplemented("Not implemented: " + this.getClass.toString)
+    (authenticate andThen getData(mode, srn) andThen allowAccess(srn) andThen requireData).async {
+      implicit request =>
+        implicit val userAnswers: UserAnswers = request.userAnswers
```

Comment:
We don't seem to need to have this as an implicit value. Also, it only seems to be used on the line below so I'm not sure it's necessary

Reply:
Its being used by the id below

Reply:
Right makes sense, didn't spot that before 👍